src/pages/proposals/special-project-details.js

```diff
@@ -1,16 +1,108 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { DetailsContainer, Content, SubTitle } from './style';
+import DigixDAOConfigChange02 from '@digix/gov-ui/assets/images/DigixDAOConfigchange-02-web.png';
+import DigixDAOConfigChange01 from '@digix/gov-ui/assets/images/DigixDAOConfigchange-01-web.png';
+import { DetailsContainer, Content, Title, SubTitle } from './style';
 
 const SpecialProjectDetails = props => (
   <DetailsContainer>
-    <Content>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-      labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
-      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
-      voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
-      non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    <Content special>
+      <Title>
+        Title: <span>Configuration change - Shortening Voting Durations for voting rounds</span>
```

Comment:
Title: <span>Configuration change - Shortening Voting Duration for voting rounds & Increasing maximum hard-cap on Ether funding per project</span>